docs/guides/page-protection.md

```diff
@@ -9,17 +9,17 @@ As a merchant, you intend your valuable content to reach, engage, and retain an
 
 ## Creating a Merchant Account
 
-In order to protect your monetization page, first, we need you to create your merchant account. For step-by-step instructions, please refer to the following [guide](https://inplayer.com/docs/getting-started/creating-your-account/).
+In order to protect your monetization page, first, we need you to create your merchant account. For step-by-step instructions, please refer to the following [guide](https://client.support.inplayer.com/getting-started/creating-your-inplayer-account/).
 Once you are successfully logged in, you would be registered in the InPlayer system as a merchant.
 
 ## Creating Your Asset
 
 Our platform supports almost all forms of online content: video, audio, text, HTML, file etc. out of which you can choose the one that best aligns with your needs. 
-To begin with, you need to log into the [InPlayer dashboard](https://dashboard.inplayer.com/dashboard) and click the 'Assets' icon from the navigation bar on the left. Then, click the '+Add new asset' button. This opens a window displaying the asset types. For the purpose of this guide let's choose the HTML/Text type. 
+To begin with, you need to log into the [InPlayer dashboard](https://dashboard.inplayer.com/dashboard) and click the 'Assets' icon from the navigation bar on the left. Then, click the '+Add new asset' button. This opens a window displaying the asset types. For the purpose of this guide let's choose the HTML/Text type paid asset. 

```

Comment:
Added 'paid asset' to the sentence because on line 22 the guide link is for paid asset. Previously the link was just HTML asset.